lib/protocol/operator/002/operator_002.ts

```diff
@@ -119,7 +120,9 @@ export class SNProtocolOperator002 extends SNProtocolOperator001 {
     }
     const localAuthHash = await this.crypto.hmac256(ciphertextToAuth, authKey);
     if (this.crypto.timingSafeEqual(authHash, localAuthHash!) === false) {
-      console.error('Auth hash does not match, returning null.');
+      const error = Error('Auth hash does not match.');
+      SNErrors.onUnhandledError(error);
+      console.error(error);
```

Comment:
Should we override console.error to call SNErrors?

Reply:
Why we would want to do that? console.errors are only for development